src/application/service/users/BulkCreateUsers.ts

```diff
@@ -16,7 +19,7 @@ export class BulkCreateUsers {
 
     for (const userCommand of createUsersCommand) {
       const authenticationDetails = new AuthenticationDetails(
-        AuthenticationMethod.fromString(userCommand.authenticationDetails.method),
+        authenticationMethodFromString(userCommand.authenticationDetails.method),
```

Comment:
What did we gain here?

Reply:
Long story, but the way the enum method was done meant we could not longer iterate over just its keys without typechecking the keys.. This meant we just made it a separate function instead.